source/unified-test-format/unified-test-format.rst

```diff
@@ -1465,20 +1468,18 @@ The following arguments are supported:
 
 - ``session``: Required string. See `commonOptions_session`_.
 
-  The client entity associated with this session SHOULD specify true for
```

Comment:
This first paragraph doesn't belong here IMO because it's talking about options for a client entity. I moved it up to where the `useMultipleMongoses` option for client entities is described.